Update the first pages with a new image and restructure them (#161)
neo4j/docs-getting-started: modules/ROOT/pages/get-started-with-neo4j/index.adoc

= Get started with Neo4j
:description: This section gives an orientation on how to get started with Neo4j.

There are a number of options on how to get started with Neo4j, and your choice depends upon your needs and goals.
Each option allows you to explore different possibilities of the Neo4j ecosystem.

[TIP]
====

> For all options on how to get started with Neo4j, see link:https://neo4j.com/try-neo4j/[].

Reviewer: ```suggestion
For all available options on how to get started with Neo4j, see link:https://neo4j.com/try-neo4j/[].
```

====

[[neo4j-first-steps]]

> == First steps with Neo4j

Author: I like this idea, so I'll make a new list more structured

* link:https://neo4j.com/cloud/platform/aura-graph-database/[Neo4j AuraDB] is a fully managed cloud service, good start for everyone who is interested in graph technologies. +
Besides the free option, you can select the subscription plan which suits you best. +

> At the moment, a few example datasets are provided on AuraDB to help you learn more about graph databases and graph query language -- Cypher.

Reviewer: ```suggestion
At the moment, AuraDB provides a few example datasets to help you learn more about graph databases and graph query language, _Cypher_.
```

+
If you are a data scientist, you might be interested in link:https://neo4j.com/cloud/platform/aura-graph-data-science/[Neo4j AuraDS] -- graph data science platform as a fully managed cloud service.
You get access to more than 65 pretuned link:https://neo4j.com/docs/graph-data-science/current/algorithms/[graph algorithms].
+
* If you want to learn more about Neo4j graph database and Cypher queries applied to a specific use case, launch link:https://login.neo4j.com/u/login?state=hKFo2SBtblVTM0tSUjFTbGRzS05xN250dXVYVk15VkNtc2drU6Fur3VuaXZlcnNhbC1sb2dpbqN0aWTZIGJiaHFuSllVTTR1VERpelV6ZEg2cUxEeU0xQ3dwVVo3o2NpZNkgRHhobWlGOFRDZXpuSTdYb2kwOFV5WVNjTEdabms0a2U[Neo4j Sandbox], where you find a number of example datasets based on different use cases.
+

> All databases are grouped into two categories:

Author: databases (I think)

> All databases are grouped into two categories:

Reviewer: They're referred to as datasets here: https://sandbox.neo4j.com/ so I'd recommend to use datasets here as well.

> All databases are grouped into two categories:

Reviewer: Also, the link doesn't work.

:description: This section gives an orientation on how to get started with Neo4j.

There are a number of options on how to get started with Neo4j, and your choice depends upon your needs and goals.
Each option allows you to explore different possibilities of the Neo4j ecosystem.

[TIP]
====
For all available options on how to get started with Neo4j, see link:https://neo4j.com/try-neo4j/[].
====

[[neo4j-first-steps]]
== First steps with Neo4j

* link:https://neo4j.com/cloud/platform/aura-graph-database/[*Neo4j AuraDB*] is a fully managed cloud service and a good place to start for anyone interested in graph technologies. +
Besides the free option, you can select the subscription plan that suits you best. +
At the moment, AuraDB provides a few example datasets to help you learn more about graph databases and graph query language, _Cypher_.
+
If you are a data scientist, you might be interested in link:https://neo4j.com/cloud/platform/aura-graph-data-science/[Neo4j AuraDS], a graph data science platform as a fully managed cloud service.
You get access to more than 65 pretuned link:https://neo4j.com/docs/graph-data-science/current/algorithms/[graph algorithms].
+

> * link:https://login.neo4j.com/u/login?state=hKFo2SBtblVTM0tSUjFTbGRzS05xN250dXVYVk15VkNtc2drU6Fur3VuaXZlcnNhbC1sb2dpbqN0aWTZIGJiaHFuSllVTTR1VERpelV6ZEg2cUxEeU0xQ3dwVVo3o2NpZNkgRHhobWlGOFRDZXpuSTdYb2kwOFV5WVNjTEdabms0a2U[*Neo4j Sandbox*] provides a number of example databases that can help you to learn more about Neo4j graph database and Cypher queries applied to a specific use case.

Reviewer: ```suggestion
* link:https://login.neo4j.com/u/login?state=hKFo2SBtblVTM0tSUjFTbGRzS05xN250dXVYVk15VkNtc2drU6Fur3VuaXZlcnNhbC1sb2dpbqN0aWTZIGJiaHFuSllVTTR1VERpelV6ZEg2cUxEeU0xQ3dwVVo3o2NpZNkgRHhobWlGOFRDZXpuSTdYb2kwOFV5WVNjTEdabms0a2U[*Neo4j Sandbox*] provides a number of example datasets that can help you to learn more about Neo4j graph database and Cypher queries applied to a specific use case.
```
They're referred to as datasets here: https://sandbox.neo4j.com/ so I'd recommend using datasets here as well. Also, the link leads to a 404.

+
All databases are grouped into two categories:
+
** for developers
** for data scientists

+
You can also select a blank sandbox and populate it with your own data.
+
* link:https://neo4j.com/product/developer-tools/[*Neo4j Desktop*] is one of the ways to set up an environment for developing an application with Neo4j and Cypher.
Download Neo4j Desktop from link:https://neo4j.com/download/[] and follow the installation instructions for your operating system. +
You get the opportunity to try all the graph applications, which are available in our link:https://install.graphapp.io/[**Graph Apps Gallery**]. +

[TIP]
====
If you need an Enterprise Edition, visit link:https://neo4j.com/pricing/#graph-database/[the Neo4j Pricing page] where you can get all necessary instructions and contacts.
====

[[neo4j-docs]]

> == Neo4j documentation page

Reviewer 2: Just a comment unrelated to this PR - this title looks odd. I would suggest something simpler
```suggestion
== Neo4j documentation
```
